### DISCUSSION TITLE
Network Issue

### DISCUSSION THREAD

**Discussion by RadoslavL** [qutebrowser/qutebrowser#6789] [General] [answered]
**Version info**:

qutebrowser v2.3.1
Git commit: 
Backend: QtWebEngine 5.15.6, based on Chromium 87.0.4280.144
Qt: 5.15.2

**Does the bug happen if you start with `--temp-basedir`?**:
 Yes, it does.
**Description**
When i wait a little not using the browser, the internet just stops working and i have to reconnect using networkmanager to use the internet again. It stops only in the browser, anything other than it seems to be able to use the internet.
**How to reproduce**

**Comment by The-Compiler** [comment]
What does "the internet stops working in the browser" mean precisely?

**Comment by RadoslavL** [comment]
When I am using a lot of internet on another tab it just disables the internet on other tabs and even windows. The only way to fix it is by closing the tab or by disconnecting from the network and reconnecting

**Comment by The-Compiler** [comment]
But what do you mean with "disables the internet on other tabs"? What exactly happens? Do you get any error messages?

**Comment by RadoslavL** [comment]
Nope. No error messages and it just happens so much. If I have a tab with a lot of network usage at some point the internet on other tabs just stops. When I try to go to a website, it shows an "No internet" error.

**Comment by The-Compiler** [comment]
All the networking stuff is handled by the underlying QtWebEngine/Chromium, so that seems like a problem on your end rather than something in qutebrowser. You'd probably see the same issues with something like KDE Falkon or even Chromium. Converting this to a discussion - maybe someone has an idea, but I doubt there's anything qutebrowser can do.

**Comment by RadoslavL** [comment, marked as answer]
It actually is a problem with the network driver, but happens quite randomly and is fixed when reinstalling the network driver. Thank you @The-Compiler for trying to find out what's the issue, but as it turns out it is not in fact an issue with qutebrowser itself.

**Reply by The-Compiler** [reply]
Thanks for the update!
